### ISSUE TITLE
In JSX, some identifiers are processed twice

### ISSUE THREAD

**Issue by spicyj** [facebook/jscodeshift#36]
…or something.

With input

```
<TAOSchemaDismissableNUX
  context={React.findDOMNode(this.refs.taolinks.refs.delete)}>
</TAOSchemaDismissableNUX>
```

this script

```
module.exports = function(file, api) {
  var j = api.jscodeshift;
  console.log(
    j(file.source)
      .find(j.Identifier, {name: 'React'})
      .size()
  );
};
```

prints 2 which seems wrong.

**Comment by fkling** [comment]
This is actually an issue with ast-types.

**Comment by fkling** [comment]
This was fixed in ast-types 0.8.11 and recast 0.10.32. Reinstalling jscodeshift should suffice. I updated http://felix-kling.de/esprima_ast_explorer/.

**Comment by spicyj** [comment]
Thanks @fkling and @benjamn!
